.mergify.yml

```diff
@@ -24,7 +24,7 @@ pull_request_rules:
         rebase_fallback: none
   - name: L10N - Auto Merge
     conditions:
-      - author=mozilla-l10n-automation-bot
+      - author~=(mozilla-l10n-automation-bot|github-actions\[bot\])
```

Comment:
This bot is also used on in fenix: https://github.com/mozilla-mobile/fenix/pull/26036